modules/ROOT/pages/cloudhub-deployment-mmp-reference.adoc

```diff
@@ -66,8 +66,8 @@ Accepted values are:
 
 Default value is  us-east-1.
 | businessGroup | Business group of the deployment.
-//_TODO: validate this
-// | server | Maven server with Anypoint Platform credentials. This is only needed if you want to use your credentials stored in your Maven settings.xml file. This is not the Mule server name.
+
+| server | Maven server with Anypoint Platform credentials. This is only needed if you want to use your credentials stored in your Maven settings.xml file. This is not the Mule server name.
```

Comment:
style tip: please put backticks around file names, e.g.: 
settings.xml -> `settings.xml` 
to change the font